src/main/java/emu/grasscutter/scripts/ScriptLibHandler.java

```diff
@@ -1473,6 +1473,7 @@ public int SetPlatformRouteId(GroupEventLuaContext context, int entityConfigId,
         }
 
         configRoute.setRouteId(routeId);
+        entityGadget.startPlatform();
```

Comment:
Are you sure setting the route should always start it? Because there are a few cases of SetPlatformRouteId with a StartPlatform afterwards, I don't think this should be the case.

Reply:
Yep! Took some figuring, but I got it to have the same functionality without doing that.